test/core/libp2p/crypto/keys_test.cpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <outcome/outcome.hpp>
+
+#include "common/buffer.hpp"
+
+#include "libp2p/crypto/public_key.hpp"
+#include "libp2p/crypto/public_key.hpp"
+#include "libp2p/crypto/common.hpp"
+
+using kagome::common::Buffer;
+using namespace libp2p::crypto; // NOLINT
```

Comment:
no need of NOLINT in tests, they are not checked.

Reply:
Ok